pkg/loop/internal/pb/relayer.proto

```diff
@@ -256,6 +256,34 @@ message FromAccountReply {
   string Account = 1;
 }
 
+service ChainReader {
```

Comment:
This file is starting to collect lots of different small services. Is it time to move components into separate files? eg. PipelineRunner and Telemetry are already in their own file; it feels like ChainReader could be too?